advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -175,15 +174,13 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
     // single element / scoring system. So, picking first severity is fine, in particular because ORT only supports a
     // single severity representation.
     var (scoringSystem, severity) = this.severity.firstOrNull()?.let {
-        Cvss.fromVector(it.score)?.let { cvss ->
-            val scoringSystem = when {
-                // Work around for https://github.com/stevespringett/cvss-calculator/issues/56.
-                it.score.startsWith("CVSS:") -> it.score.substringBefore("/")
-                cvss is CvssV2 -> "CVSS:2.0"
-                else -> cvss.vector.substringBefore("/", "CVSS")
-            }
+        require(it.type == Severity.Type.CVSS_V3) {
```

Comment:
Commit message:
- "as a / the scoring system"
- put `requires` in backticks for code formatting
- "rather" -> "other"